framework/platform/bpl/include/bpl/bpl_cfg.h

```diff
@@ -225,6 +225,12 @@ struct BPL_WLAN_PARAMS {
     char security[BPL_SEC_LEN];
 };
```

Comment:
Commit message headline should be "bpl: add cfg_get_all_prplmesh_wifi_interfaces API"
What is "prpl" interfaces"? don't use this terminology.